test/e2e/package.json

```diff
@@ -10,9 +10,7 @@
   },
   "type": "module",
   "dependencies": {
-    "@thirdweb-dev/sdk": "^4.0.99",
-    "ethers": "5",
-    "thirdweb": "^5.43.2",
-    "viem": "^2.18.1"
```

Comment:
We want to have tests that turn off interval mining on the chain to simulate features like the chain stopping producing blocks etc, and [viem test client](https://viem.sh/docs/actions/test/mine.html) is helpful there.

Reply:
I think you can configure that with prool, but can always add back!